src/apps/routes/apps.clj

```diff
@@ -1,29 +1,86 @@
 (ns apps.routes.apps
   (:use [common-swagger-api.routes]
         [common-swagger-api.schema]
-        [common-swagger-api.schema.apps :only [AppDeletionRequest
-                                               AppListing
-                                               AppListingSummary
-                                               AppsShredderDocs
-                                               AppsShredderSummary
-                                               SystemId]]
-        [common-swagger-api.schema.apps.permission :only [AppPermissionListing
-                                                          AppPermissionListingDocs
-                                                          AppPermissionListingRequest
-                                                          AppPermissionListingSummary
-                                                          AppSharingDocs
-                                                          AppSharingRequest
-                                                          AppSharingResponse
-                                                          AppSharingSummary
-                                                          AppUnsharingDocs
-                                                          AppUnsharingRequest
-                                                          AppUnsharingResponse
-                                                          AppUnsharingSummary]]
+        [common-swagger-api.schema.apps
+         :only [App
```

Comment:
I moved the `:only` list to a new line since I thought it looked a little better with these longs lists.
I can reformat these lists with the `:only` on the same line as the namespace if everyone else prefers that style.

Now that I think about it, it might even be better to require a namespace alias (like `common-swagger-api.schema.apps  :as apps-schema`) and use that instead of these long `:use ... :only` lists 🤔

Reply:
Actually, I've convinced myself that I prefer the `[common-swagger-api.schema.apps :as schema]` alias, so I'll push a commit that removes this `:only` list.

I'm leaving the `common-swagger-api.schema.apps.permission :only` list below, since it's short enough, but I can still update its style if preferred.